inc/HTTP/HTTPClient.php

```diff
@@ -749,7 +749,7 @@ protected function debugText($info, $var=null){
     /**
      * convert given header string to Header array
      *
-     * All Keys are lowercased.
+     * All Keys are in lowercase.
```

Comment:
this is not correct. The method converts keys into lower case. This meaning is lost by your change.

Reply:
Removed 4a333d73cdeafaf0fc87d9b0253c65b550de53a9